articles/openshift/oshift-how-restrict-access-to-openshift-routes-by-ip.md

```diff
@@ -1,57 +1,65 @@
 ---
-title: UKCloud Knowledge Centre article about whitelisting public IP's on your openshift routes | UKCloud Ltd
-description: Restrict access to openshift routes by public IP[s]
+title: How to restrict access to a route in OpenShift by the source public IP address
+description: Restrict access to OpenShift routes by the source public IP address
 services: openshift
 author: Mudasar Hussain
 toc_rootlink: How To
 toc_sub1: 
 toc_sub2:
 toc_sub3:
 toc_sub4:
-toc_title: restrict access to openshift routes by public IP[s]
+toc_title: Restrict access to OpenShift routes by the source public IP address
 toc_fullpath: How To/restricting-access-to-openshift-routes-by-ip.md
 toc_mdlink: oshift-how-restrict-access-to-openshift-routes-by-ip.md
 ---
 
-# How to restrict access to a route in openshift by IP[s] address
+# How to only allow trusted IP's to be able to access routes in Openshift
```

Comment:
Could you make sure that this heading and the title in the metadata are the same